src/qtoolkit/io/sge.py

```diff
@@ -0,0 +1,526 @@
+            qjob = QJob()
+            qjob.job_id = self._get_element_text(job_element, "JB_job_number")
+            job_state_string = self._get_element_text(job_element, "state")
+
+            try:
+                sge_job_state = SGEState(job_state_string)
+            except ValueError:
+                raise OutputParsingError(
+                    f"Unknown job state {job_state_string} for job id {qjob.job_id}"
+                )
+
+            qjob.sub_state = sge_job_state
+            qjob.state = sge_job_state.qstate
+            qjob.username = self._get_element_text(job_element, "JB_owner")
+            qjob.name = self._get_element_text(job_element, "JB_name")
+
+            info = QJobInfo()
+            info.nodes = self._safe_int(
+                self._get_element_text(job_element, "num_nodes")
+            )
+            info.cpus = self._safe_int(self._get_element_text(job_element, "num_proc"))
+            info.memory_per_cpu = self._convert_memory_str(
+                self._get_element_text(job_element, "hard resource_list.mem_free")
+            )
+            info.partition = self._get_element_text(job_element, "queue_name")
+            info.time_limit = self._convert_str_to_time(
+                self._get_element_text(job_element, "hard resource_list.h_rt")
+            )
+
+            qjob.info = info
+
+            jobs_list.append(qjob)
+
+        return jobs_list
+
+    @staticmethod
+    def _convert_str_to_time(time_str: str | None) -> int | None:
+        if time_str is None:
+            return None
+
+        parts = time_str.split(":")
+        if len(parts) == 3:
+            hours, minutes, seconds = parts
+        elif len(parts) == 2:
+            hours, minutes = "0", parts[0]
+            seconds = parts[1]
+        elif len(parts) == 1:
+            hours, minutes, seconds = "0", "0", parts[0]
+        else:
+            raise OutputParsingError(f"Invalid time format: {time_str}")
+
+        try:
+            return int(hours) * 3600 + int(minutes) * 60 + int(seconds)
+        except ValueError:
+            raise OutputParsingError(f"Invalid time format: {time_str}")
+
+    @staticmethod
+    def _convert_memory_str(memory: str | None) -> int | None:
+        if not memory:
+            return None
+
+        match = re.match(r"([0-9]+)([a-zA-Z]*)", memory)
+        if not match:
+            raise OutputParsingError("No numbers and units parsed")
+        memory, units = match.groups()
+
+        power_labels = {"K": 0, "M": 1, "G": 2, "T": 3}
+
+        if not units:
+            units = "M"
+        elif units not in power_labels:
+            raise OutputParsingError(f"Unknown units {units}")
+        try:
+            v = int(memory)
+        except ValueError:
+            raise OutputParsingError
+
+        return v * (1024 ** power_labels[units.upper()])
+
+    _qresources_mapping = {
+        "queue_name": "queue",
+        "job_name": "job_name",
+        "account": "account",
```

Comment:
There is no `account` value in the template defined above. According to [this link](https://gridscheduler.sourceforge.net/htmlman/htmlman1/qsub.html) the `-A` is available in the `qsub` command for SGE as well, but on other websites not. Can you confirm this? And if true add it to the template like in the case of PBS?
https://github.com/Matgenix/qtoolkit/blob/df856926877b1c7f038919887799e17241d53f07/src/qtoolkit/io/pbs.py#L86
Otherwise remove it from here.

Reply:
I think I can simply remove it, if needed it can still be added via verbatim command I guess.